src/store/actions.js

```diff
@@ -28,19 +28,6 @@ export function init() {
                       edited: false
                     }));
       dispatch(start(result));
-
-      // console.log(response.data.slice(0,15));
-      // dispatch({
-      //   type: INIT,
-      //   payload: response.data.slice(0, 15).map(card => ({
-      //     id: card.Number,
-      //     caption: card.Name,
-      //     text: card.About,
-      //     checked: false,
-      //     edited: false
-      //   }))
-      // })
-
```

Comment:
Ideally there should be no commented code. If you require any artifact code to be stored at some point - you can save it as a part of some intermediate commit. But final commit should have it removed.